src/App.tsx

```diff
@@ -2,15 +2,35 @@ import React from 'react';
 
 import './App.scss';
 
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
+import { User } from './types/User';
+import { Post } from './types/Post';
+import { Comment } from './types/Comment';
+import { PostList } from './components/PostList';
+
+function getComment(postId:number): Comment[] {
```

Comment:
![image](https://github.com/mate-academy/react_static-list-of-posts/assets/96409563/78b59373-c9d7-4fb7-91d7-b6a0b1d2a0ea)